workos/resources/directory_sync.py

```diff
@@ -1,4 +1,9 @@
+import workos
 from workos.resources.base import WorkOSBaseResource
+from workos.utils.request import (
```

Comment:
Are these imports necessary? They don't seem to be required for the addition of `primary_email`.